xml/ha_storage_protection.xml

```diff
@@ -842,21 +793,128 @@ SBD_WATCHDOG_TIMEOUT=5</screen>
 
    <para> After the resource has started, your cluster is successfully
     configured for use of SBD and will use this method in case a
-    node needs to be fenced. </para>
+    node needs to be fenced.</para>
+  </sect1>
+
+  <sect1 xml:id="sec.ha.storage.protect.diskless-sbd">
+   <title>Setting Up Diskless SBD</title>
+   <para>SBD can be operated in a diskless mode. In this mode, a watchdog device
+    will be used to reset the node in the following cases: if it loses quorum,
+    if any monitored daemon is lost and not recovered or if Pacemaker decides
+    that the node requires fencing. Diskless SBD is basically based on
+     <quote>self-fencing</quote> of a node, depending on the status of the quorum
+     and some reasonable assumptions.</para>
+    <important>
+     <title>Number of Cluster Nodes</title>
+       <para>
+         Do <emphasis>not</emphasis> use diskless SBD as fencing mechanism
+         for two-node clusters. Use it only in clusters with three or more
+         nodes. SBD in diskless mode cannot handle split brain
+         scenarios for two-node clusters.
+      </para>
+   </important>
+
+   <procedure xml:id="pro.ha.storage.protect.confdiskless">
+    <title>Configuring Diskless SBD</title>
+    <step>
+     <para>Open the file <filename>/etc/sysconfig/sbd</filename> and use
+      the following entries:</para>
+     <screen>SBD_PACEMAKER=yes
+SBD_STARTMODE=always
+SBD_DELAY_START=no
+SBD_WATCHDOG_DEV=/dev/watchdog
+SBD_WATCHDOG_TIMEOUT=5</screen>
+      <para>
+       The <varname>SBD_DEVICE</varname> entry is not needed as no shared
+       disk is used. When this parameter is missing, the <systemitem>sbd</systemitem>
+       service does not start any watcher process for SBD devices.
+      </para>
+    </step>
+    <step>
+     <para>On each node, enable the SBD service to start at boot time: </para>
```

Comment:
enable the SBD service to start when starting cluster stack

-- sbd.service has this special configuration "PartOf=corosync.service". So enabling sbd.service doesn't necessarily means it'll start at boot time. It only means it'll start together with corosync.service. They are actually brought up by pacemaker.service by dependencies.

Reply:
Changed in all places in the chapter.